feature-libs/asm/components/services/asm-component.service.ts

```diff
@@ -39,6 +42,14 @@ export class AsmComponentService {
     this.isEmulatedByDeepLink$.next(emulated);
   }
 
+  setShowInactiveCartInfoAlert(display: boolean) {
+    this.showInactiveCartInfoAlert$.next(display);
+  }
+
+  showInactiveCartInfoAlert() {
```

Comment:
provide return value in func declaration

Reply:
fixed